src/services/staff.js

```diff
@@ -0,0 +1,8 @@
+import { client } from '@/api/axiosClient';
+
+export function registerStaff({ name, surname, email, password, specialty, clinicId, dni }) {
+  return client.post(
+    '/staff/register',
+    { name, surname, email, password, specialty, clinicId, dni }
+  );
+}
```

Comment:
clinicId is taken but never provided. It is not part of the schema

Reply:
Resolved in https://github.com/FIS2425/frontend/pull/30/commits/fc5b6a5a8fc932cadd0de18f3930c4b7bd1294a4